graph-platform/neo4j-desktop/neo4j-desktop.adoc

```diff
@@ -100,38 +109,52 @@ A panel will display on the right to show the databases in your project and the
 
 To create a database, click in the dotted square that says `Add Graph`.
 Then click the kbd:[Create a Local Graph] button.
-Next, you will type in a name for your database and set a password and choose a version (default is latest version).
-A small icon will show which versions have been downloaded already.
+Next, you will type in a name for your database set a password.  You can also choose a specific version of neo4j (the default is latest version).
+A small icon next to each version number in the dropdown menu will show which versions have been downloaded already.
 Once the info is filled in, then click kbd:[Create].
 It may take a few minutes to download the version and actually create the database.
 
 ****
 [NOTE]
-Please remember your database password. 
+Please remember your database password.
 If you forget it, you will have to remove the credentials file in `data/dbms/authorization`.
```

Comment:
Override password in the Administration pane instead.

Reply:
Also as most graph apps work automatically without the password, this is only relevant for driver access, so perhaps we can deemphasise?